stdlib/influxdata/influxdb/influxdb.flux

```diff
@@ -6,7 +6,15 @@
 package influxdb
 
 
-// cardinality returns the series cardinality of data stored in InfluxDB.
+// cardinality returns the series cardinality of data retrieved InfluxDB.
+//
+// Although this function is similar to InfluxQL's `SHOW SERIES CARDINALITY` it works in a slightly
+// different manner.
+//
+// cardinality is time bounded and reports the  `influxdb.cardinality()` is time bounded and reports
```

Comment:
I think you've doubled up this sentence a little.

Reply:
Doh. ta!